src/components/App.js

```diff
@@ -17,14 +19,35 @@ class App extends Component {
     const questionsUrl = () => `http://localhost:3001/quizzes`;
     return fetch(questionsUrl())
     .then(result => result.json())
-    .then(jsonResult => console.log(jsonResult))
+    .then(result => {this.setState({
+      quizzes: result.quizzes,
+      questions: result.quizzes[0].questions,
+    });
+      console.log(result);
+    })
     .catch(error => console.log(error));
   }
 
   render() {
     return (
       <div className="App">
-        <h1>What JavaScript Error Are You?</h1>
+        {this.state.quizzes.map(quiz => { return (
+            <div key={quiz.id}>
+              <h1>{quiz.title}</h1>
+            </div>
+          )}
+        )}
+        {this.state.questions.map(question => { return (
```

Comment:
Is it better to have the Questions / Answers components be stateful or stateless? I'm having trouble with the state/ props I'm passing my dumb component being undefined on load. I'm even trying to use an if statement to try to guard against my render method being undefined.